### PR TITLE
Arrival date improvement

### PR DESCRIPTION
**Pull request by victor-paumier**
- add tooltip on semester
- add profile complete column on list users
- add match button on list users
- allow BCs to edit semester & arrival date

### CODE REVIEW

File: src/Entity/User.php [hunk 1]
```diff
@@ -698,18 +698,23 @@ public function getPictureName(): ?string
         return $this->pictureName;
     }
 
-    public function isFullyCompleted(): bool
+    public function isCompleteForMatching(): bool
```
Comment (JeremieSamson) [inline src/Entity/User.php]: What of isReadyForMatching ?

File: src/Model/UserListResponse.php [hunk 2]
```diff
@@ -27,6 +27,7 @@ class UserListResponse implements DataTableResponseInterface
     private $actions;
     private $DT_RowAttr;
     private $sectionName;
+    private $isProfileComplete;
```
Comment (JeremieSamson) [inline src/Model/UserListResponse.php]: What of $isProfileCompleted

File: templates/Default/listActions.html.twig [hunk 3]
```diff
@@ -8,31 +8,39 @@
...
+        {% if urlTranslate is defined and urlTranslate is not null%}
             <li>
                 <a href="{{ urlTranslate }}"><i class="fas fa-language"></i>
                     {{ 'actions.translate'|trans }}
                 </a>
             </li>
         {% endif %}
 
-        {% if urlShow is defined %}
+        {% if urlShow is defined and urlShow is not null %}
```
Comment (JeremieSamson) [inline templates/Default/listActions.html.twig]: last time I told you to do a function in twig to do an isset, you told me you only used it once, well ... what do you think of using it now ?

Comment (victor-paumier) [reply templates/Default/listActions.html.twig]: what about `is not empty`?

File: src/Entity/User.php (hunk 1 above)
Comment (victor-paumier) [reply src/Entity/User.php]: done

File: src/Model/UserListResponse.php (hunk 2 above)
Comment (victor-paumier) [reply src/Model/UserListResponse.php]: done

File: templates/Default/listActions.html.twig (hunk 3 above)
Comment (victor-paumier) [reply templates/Default/listActions.html.twig]: How do you do that? I got an error when I try to pass and undefined var to a custom twig function

**Pull request merged**